File: tracks/python/difference_of_squares.py
def sequence(number):
    numbers = []
    for i in range(number + 1):
        numbers.append(i)
    return numbers


def square_of_sum(number):
    sequence_of_numbers = sequence(number)
    return sum(sequence_of_numbers) ** 2


def sum_of_squares(number):
    sequence_of_numbers = sequence(number)
    result = 0
    for n in range(number + 1):
        result += n ** 2
    return result


def difference_of_squares(number):
    return square_of_sum(number) - sum_of_squares(number)

File: tracks/javascript/difference-of-squares.js
export class Squares {
    constructor(n) {
        this.number = n;
    }
  
    get sumOfSquares() {
        let sum = 0;
        for (let i = 1; i <= this.number; i++) {
            sum += i*i;
        }
        return sum;
    }
  
    get squareOfSum() {
        let sum = 0;
        for (let i = 1; i <= this.number; i++) {
            sum += i;
        }
        return sum * sum;        
    }
  
    get difference() {
        return this.squareOfSum - this.sumOfSquares;
    }
}